content/docs/setup/kubernetes/sidecar-injection.md

```diff
@@ -7,16 +7,6 @@ aliases:
     - /docs/setup/kubernetes/automatic-sidecar-inject.html
 ---
 
-> The following requires Istio 0.5 or greater. See
-> [https://archive.istio.io/v0.4/docs/setup/kubernetes/sidecar-injection](https://archive.istio.io/v0.4/docs/setup/kubernetes/sidecar-injection)
-> for Istio 0.4 or prior.
->
-> In previous releases, the Kubernetes initializer feature was used for automatic proxy injection. This was an Alpha feature, subject to change/removal,
-> and not enabled by default in Kubernetes. Starting in Kubernetes 1.9 it was replaced by a beta feature called
-> [mutating webhooks](https://kubernetes.io/docs/admin/admission-controllers/#mutatingadmissionwebhook-beta-in-19), which is now enabled by default in
-> Kubernetes 1.9 and beyond. Starting with Istio 0.5.0 the automatic proxy injection uses mutating webhooks, and support for injection by initializer has been
-> removed. Users who cannot upgrade to Kubernetes 1.9 should use manual injection.
```

Comment:
nice 👍 For further reviews, has a decision been made on a minimum pin version of Kubernetes?  I saw a different review which changed the tested version to 1.10.